ui/components/atoms/A007 - Custom Buttons with CSS/index.js

```diff
@@ -0,0 +1,80 @@
+// List Libraries being used
+import React from 'react'
+import createPropTypes from 'json-schema-prop-types'
+import { withStyles } from '@material-ui/core/styles'
+
+// Import from UI library
+import Button from '@material-ui/core/Button'
+
+// Import JSON schema
+import schema from './schema'
+
+// Import styles here - see example setup for styles at ui/example/components/atoms/checkbox
+
+// custom styling
+
+// 1. CSS Stylesheet
+import './custom.css';
+
+  // 2. Inline styling
+  const styles = {
+    root: {
+      backgroundColor: 'white',
+      border: '5px solid #eee',
+      marginTop: '20px',
+      padding: '20px'
+    },
+
+    buttonStyle: {
+      backgroundColor: 'white',
+      color: 'red',
+      margin: '5px',
+    }
+  };
+
+// Define Atom as JSX
+const A007 = props => {
+  
+  const { classes } = props;
+
+  return (
+    <div className={classes.root}>
```

Comment:
atoms dont have divs in them - move all this to a story

Reply:
I removed this whole A007